packages/vm/src/eei/eei.ts

```diff
@@ -12,7 +12,7 @@ type Block = {
 }
 
 type Blockchain = {
-  getBlock(blockId: number): Promise<Block | null>
+  getBlock(blockId: number): Promise<Block>
```

Comment:
I just changed this to align, not 100% sure if this might have some effects somewhere.

Generally seems to me the right thing to do.

Reply:
This is what I originally wanted to do so glad we're able to incorporate this.  I'm assuming this does indeed make this breaking change?

Reply:
I actually don't think that this is so breaking, having dealt with similar code parts. If someone has prepared for dealing with Block or `null` in his/her code base, all well, now it's just limited to only Blocks. Especially regarding the fact that `null` de facto never was a valid option respectively never occurred.

So I think we can label this as a bugfix and pack this in at most a minor release version.